controller.js

```diff
@@ -65,3 +66,14 @@ exports.getReviews = (req, res, next) => {
       next(err);
     });
 };
+
+exports.getComments = (req, res, next) => {
+  const { review_id } = req.params;
+  fetchComments(review_id)
```

Comment:
in order to check for a valid `review_id` , we can call the model function `fetchReviewByID` , and use the pre-written error handlers on that model to make sure it's valid first, then call upon `fetchComments` afterwards